documentation/how-to-use-appyx/sample-apps.md

```diff
@@ -12,7 +12,13 @@ The `:app` module showcases **Appyx** itself with multiple levels of navigation,
 
 ## Appyx + Jetpack Compose Navigation example 
 
-Coming soon!
+The `:samples:navigation-compose` module demonstrates how to use Appyx within Google's Jetpack Compose Navigation library.
+This example may be useful if you need to migrate to Appyx gradually.
+
+1. Go to the [GitHub project](https://github.com/bumble-tech/appyx)
```

Comment:
Perhaps we want to reduce the repetition?

Reply:
Sure. You could pull out a section to the top "## Where to find the sample apps"

Reply:
done